development/git-workflow/README.md

```diff
@@ -51,5 +51,56 @@ In all other branches we'll use a build number within a tagged commit to designa
 3. Create a commit with updated deliverables.
 4. Tag that commit with a build number.
 
+# Build Process for Creating a QA build
+
+Prerequisite: create qa branches on alpheios-core, embed-lib and webextension and push them upstream. 
+
+1. alpheios-core: merge master to qa
+2. alpheios-core: run `npm install && npm update`
+3. alpheios-core: run `npm tagged-commit`
```

Comment:
For `alpheios-core` we also need to run a `bootstrap` npm script that will install dependencies of the sub-packages.

Reply:
thanks will fix that too. We actually should run `npm bootstrap` only, correct?

Reply:
> We actually should run `npm bootstrap` only, correct?

Yes, As I side note, I had sometimes some dependencies being not present in `node_modules` after running some combination of `npm install` and `bootstrap`, but I cannot say what sequence was causing this exactly; it seems like some floating issue to me. Because we don't know the conditions we cannot put it to the docs, but we should probably keep an eye on that while working.

I'm also not sure whether we should run `npm update` for `alpheios-core`. Majority of dependencies of `alpheios-core` root `package.json` are those of `node-build`. They were set by the `set-node-build-deps` npm script. They probably should match whatever `node-build` has in its peer dependencies, but in majority of cases updated dependencies will work too (and will have bug fixes and other improvements which are useful). What do you think?

Reply:
I am not sure. I think ideally, we would do updates during development and then freeze dependencies for QA and production.  Not sure the best way to manage that though.